src/main/java/com/rocana/lucene/codec/v1/RocanaBlockTreeTermsReader.java

```diff
@@ -44,7 +45,28 @@
 import org.apache.lucene.util.fst.ByteSequenceOutputs;
 import org.apache.lucene.util.fst.Outputs;
 
-/** A block-based terms index and dictionary that assigns
+/**
+ * Fork of Lucene's {@link com.rocana.lucene.codec.v1.RocanaBlockTreeTermsReader}.
+ *
+ * Why we forked:
+ *   - We can't extend because it has final fields that must be set in the constructor,
+ *     and the constructor is the code we need to modify.
+ *
+ * What changed in the fork?
+ *   - Commented out the constructor's call to {@link CodecUtil#checksumEntireFile(IndexInput)}
+ *     for performance.
+ *   - Use the other forked classes, like {@link RocanaFieldReader}.
+ *   - Removed trailing whitespace.
+ *   - Changed these javadocs.
+ *
+ * To see a full diff of changes: compare this version to this file's initial commit (which
+ * is the original file from Lucene).
```

Comment:
should add the version and/or Lucene tag to this comment: 5.5.0, git tag `releases/lucene-solr/5.5.0`